problem with --enable-benchmarks and current master

I have not yet had time to look into this, but I am seeing this on a make -j distcheck build with --enable-benchmarks. I will take a look after I get the rest of my testing set up and running...

```
make[5]: Entering directory '/var/lib/jenkins/workspace/ejh_noaa_netcdf-c_features/feature/--enable-benchmarks/netcdf-c-4.7.3-development/_build/sub/nc_perf'
make[5]: Nothing to be done for 'run_tst_chunks.log'.
make[5]: Nothing to be done for 'run_bm_elena.log'.
make[5]: Leaving directory '/var/lib/jenkins/workspace/ejh_noaa_netcdf-c_features/feature/--enable-benchmarks/netcdf-c-4.7.3-development/_build/sub/nc_perf'
fatal: making test-suite.log: failed to create run_tst_chunks.trs
fatal: making test-suite.log: failed to create run_tst_chunks.log
fatal: making test-suite.log: failed to create run_bm_elena.trs
fatal: making test-suite.log: failed to create run_bm_elena.log
Makefile:969: recipe for target 'test-suite.log' failed
```

--- a/nc_perf/Makefile.am
+++ b/nc_perf/Makefile.am
@@ -54,8 +54,8 @@ run_par_bm_test.log: tst_create_files.log
 endif # TEST_PARALLEL4
 endif # BUILD_UTILITIES
 
-EXTRA_DIST = run_par_bm_test.sh.in run_knmi_bm.sh	\
-perftest.sh run_bm_test1.sh run_bm_test2.sh \
+EXTRA_DIST = run_par_bm_test.sh.in run_knmi_bm.sh perftest.sh		\
+run_bm_test1.sh run_bm_test2.sh run_tst_chunks.sh run_bm_elena.sh	\
 CMakeLists.txt
 
 CLEANFILES = tst_*.nc bigmeta.nc bigvars.nc floats*.nc	\

